app/views/management/index.html.erb

```diff
@@ -10,3 +10,8 @@
     <%= link_to "Index Ladybird Records to Solr", metadata_source_path("ladybird"), class: "btn btn-primary" %>
     <%= link_to "Index ArchiveSpace Records to Solr", metadata_source_path("aspace"), class: "btn btn-primary" %>
 </div>
+
+<br /> 
+
+<%= render partial: "container" %>
```

Comment:
Can we come up with something more descriptive for this partial?

Reply:
Changed `service_display`